projects/storefrontapp-e2e-cypress/cypress/e2e/b2b/regression/unit-level-order/b2b-unit-level-order-history.e2e.cy.ts

```diff
@@ -168,22 +165,6 @@ describe('B2B - Unit-Level Orders History', { testIsolation: false }, () => {
       checkThatOrdersAreSorted(fieldBuyer);
     });
 
-    it("can select 'Buyer (Descending)' to see the list sorted descending by unit name", () => {
```

Comment:
remove 2 sorted tests out of 4 to keep it short, I don't think we need to test all of them.